src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
+import { repeatArray } from '../../services/utils'
```

Comment:
![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/8972ebb2-7485-48a0-98ea-646e8f440d69)

Inactive users are not editable for some reason. (or deletable)